massa-factory-worker/src/block_factory.rs

```diff
@@ -237,9 +237,9 @@ impl BlockFactoryWorker {
         );
 
         // send full block to consensus
-        // self.channels
-        //     .consensus
-        //     .register_block(block_id, slot, block_storage, true);
+        self.channels
+            .consensus
+            .register_block(block_id, slot, block_storage, true);
```

Comment:
what happened here ?

Reply:
I think it was just during some tests, and was never meant to be commented out on testnet.